examples/designdoc.py

```diff
@@ -32,7 +33,8 @@ def main():
     parser.add_option('--ccd', dest='ccd', type=int, default=5, help='Suprimecam CCD number')
     parser.add_option('--prefix', dest='prefix', default='design-', help='plot filename prefix')
     parser.add_option('--suffix', dest='suffix', default=None, help='plot filename suffix (default: ".png")')
-    parser.add_option('--pat', dest='pat', help='Plot filename pattern: eg, "design-%(pid)04i-%(name).png"; overrides --prefix and --suffix')
+    parser.add_option('--pat', dest='pat',
+        help='Plot filename pattern: eg, "design-%(pid)04i-%(name).png"; overrides --prefix and --suffix')
```

Comment:
I think this was to limit the line length to under 110 chars. I could split the string into two lines but I think it looks better the way it is.